deployments/eecs/image/requirements.txt

```diff
@@ -18,3 +18,5 @@ jupyter-desktop-server==0.1.*
 # For EECS127/227a
 # See https://github.com/berkeley-dsep-infra/datahub/issues/1631
 cvxpy==1.1.*
+# See https://github.com/berkeley-dsep-infra/datahub/issues/1936
+sklearn==0.23.*
```

Comment:
I think the package name is `scikit-learn` (https://pypi.org/project/scikit-learn/). Can you amend the PR to match that? Thanks!

Reply:
Yes, I noticed it failed the check. Thank you!